Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -162,6 +162,12 @@ - (void)commontInit
     UILongPressGestureRecognizer *longPress =
         [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(handleLongPressGesture:)];
     [self addGestureRecognizer:longPress];
+
+    PanDirectionGestureRecognizer *panGesture =
+        [[PanDirectionGestureRecognizer alloc] initWithDirection:PanDirectionHorizontal
+                                                          target:self
+                                                          action:@selector(handlePanGesture:)];
+    [self addGestureRecognizer:panGesture];
```

Comment:
Should this GR hang on the cells or on the conversation collection view?  

* This won't work for pans that start in offers, errors, calls, unread indicator.
* This won't work for pans that start outside messages (on the left of outgoing, on the right of incoming).

Reply:
Oh I see this is to show message metadata.  Please ignore.